Modules/posixmodule.c

```diff
@@ -8779,6 +8782,14 @@ void
 _Py_closerange(int first, int last)
 {
     first = Py_MAX(first, 0);
+#ifdef HAVE_CLOSE_RANGE
+    if (close_range(first, last, 0) == -1 && errno != ENOSYS) {
```

Comment:
Maybe I am missing something but if close_range returns 0 (success) then it will fallback to other possibilities no?

Reply:
Whoops, correct... last-minute shifting of #ifdefs caused a brain-o.

Reply:
thanks pablogsal!